index.html

```diff
@@ -2138,6 +2139,75 @@ <h2>
+                  the result.
+                </li>
+                <li>
+                  <p>
+                    If <var>regionCode</var> is not a valid <a>country
+                    subdivision code element</a> as per [[!ISO3166-2]]'s
+                    section 5.2 "Structure of country subdivision code
+                    elements" (non-normative details below), throw a
+                    <a>RangeError</a> exception.
+                  </p>
+                  <div class="note" title=
+                  "Structure of country subdivision code elements">
+                    <p>
+                      <strong>Do not implement from this note.</strong> The
+                      structure of a <a>country subdivision code element</a> is
+                      formally defined in [[!ISO3166-2]] (section 5.2).
+                      Although the structure is not expected to change at the
+                      time of writing, implementers are expected to track
+                      updates to [[!ISO3166-2]] directly from ISO.
+                    </p>
+                    <p>
+                      As [[!ISO3166-2]] is not freely available to the general
+                      public, the structure of a <a>country subdivision code
+                      element</a> at the time of publication is as follows:
+                    </p>
+                    <ul>
+                      <li>Two <a>code points</a> that match an [[!ISO3166-1]]
+                      alpha-2 country code.
+                      </li>
+                      <li>A single U+002D (-) <a>code point</a>.
+                      </li>
+                      <li>One, two, or three <a data-cite=
+                      "INFRA#ascii-alphanumeric">ASCII alphanumeric</a> code
+                      points, in any order.
+                      </li>
+                    </ul>
+                  </div>
+                </li>
+                <li>Set <var>address</var>.<a>[[\regionCode]]</a> to
+                <var>regionCode</var>.
+                </li>
+                <li>Let <var>region</var> be the corresponding <a>country
+                subdivision name</a> for <var>regionCode</var>. Where
+                [[!ISO3166-2]] defines multiple <a>country subdivision
+                names</a> for a <var>regionCode</var>, it is RECOMMENDED the
+                user agent select one by matching on:
+                  <ol>
+                    <li>The <a data-cite="!HTML#language">language</a> of
+                    <a data-cite="HTML#the-body-element-2">the body
+                    element</a>.
+                    </li>
+                    <li>The user's preferred languages.
+                    </li>
+                    <li>Any other criteria the user agent deems suitable.
+                    </li>
+                  </ol>
+                </li>
+                <li>Set <var>address</var>.<a>[[\region]]</a> to
```

Comment:
@domenic, stop reading this! get back to vacation! 🍹

But when you are back... by design, `address.region` is overridable by the developer.  So, eventually:

```JS
const dict = {
   country: "PT",
   regionCode: "PT-11",
}
const address = new PaymentAddress(dict);
address.region; // "Lisboa", per ISO ISO3166-2

// And... then.. 
const dict = {
   country: "PT",
   regionCode: "PT-11",
   region: "Lisbon",
}
const address = new PaymentAddress(dict);
address.region; // Lisbon, per developer override.
```

Hmm.... now I'm wondering if we should allow nullable on the AddressInit dictionary:

```JS
const dict = {
   country: "PT",
   regionCode: "PT-11",
   region: null, // or undefined, means "derive it for me" 
}
``` 

While:
```JS
const dict = {
   country: "PT",
   regionCode: "PT-11",
   region: "", // trash it ... I'll handle it
}
```

Reply:
I think `undefined` ("not present" in dictionary-speak) is the right signifier. So basically only execute this step if `details["region"]` does not exist. Otherwise set address.[[region]] to details["region"].

Whether we want to allow details["region"] to be nullable or not, as some kind of explicit "I'm not sure.